pcs_test/tier0/common/interface/test_dto.py

```diff
@@ -28,7 +28,7 @@ def _import_all(_path):
         if module_name.startswith("_pcs.snmp."):
             continue
         del is_pkg
-        loader.find_module(module_name).load_module(module_name)
+        loader.find_spec(module_name).loader.load_module(module_name)
```

Comment:
The fix is good, but the line just became confusing because `loader` is now in it twice which gives an impression that `find_spec(module_name).loader` is redundant.  I would like you to rewrite the name of the variable returned by `walk_packages`. For example:
```suggestion
        module_finder.find_spec(module_name).loader.load_module(module_name)
```